flake.nix

```diff
@@ -141,6 +141,11 @@
         flake-compat.follows = "flake-compat";
       };
     };
+
+    kernel-hardening-checker = {
```

Comment:
Could be also integrated into the CI like this:

{
  runCommand "kernel-check" {} ''
    kernel-hardening-checker -c ${pkgs.hardened-kernel.configfile}
  '':
}

Reply:
Where the hardened-kernel is the attribute of this PR: https://github.com/tiiuae/ghaf/pull/439

Reply:
We have plan for it -- make json report from KHC, then use custom script to assert checks (because KHC asserts on too much things, we need to ignore some checks like CONFIG_MODULE)